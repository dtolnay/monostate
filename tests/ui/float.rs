use monostate::MustBe;

fn main() {
    let _ = MustBe!(1.0);
    let _ = MustBe!(b"...");
    let _ = MustBe!(c"...");
}
