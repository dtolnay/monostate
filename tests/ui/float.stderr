error: unsupported monostate literal kind
 --> tests/ui/float.rs:4:21
  |
4 |     let _ = MustBe!(1.0);
  |                     ^^^

error: unsupported monostate literal kind
 --> tests/ui/float.rs:5:21
  |
5 |     let _ = MustBe!(b"...");
  |                     ^^^^^^

error: unsupported monostate literal kind
 --> tests/ui/float.rs:6:21
  |
6 |     let _ = MustBe!(c"...");
  |                     ^^^^^^
